pkg/network/cache/infocache.go

```diff
@@ -114,6 +126,27 @@ func newPodInterfaceCacheStore(vmi *v1.VirtualMachineInstance, baseDir, pattern
 	return podInterfaceCacheStore{vmi: vmi, baseDir: baseDir, pattern: pattern}
 }
 
+type vmNetworkingInfraInterfacesStore struct {
+	pid     string
+	pattern string
+	baseDir string
+}
+
+func (d vmNetworkingInfraInterfacesStore) Read(iface string) (*networkdriver.VIF, error) {
```

Comment:
the `iface` has to be named `ifaceName` to be consistent with `Write`

Reply:
Done.